src/sass/components/_card.scss

```diff
@@ -161,18 +161,31 @@
     }
 
     &__candidate-read-more-link {
-      left: 0;
       margin-left: 10px;
       top: 0;
-      font-weight: lighter;
+      font-family: $body-font-stack;
+      font-size: 14px;
+      font-weight: 100 !important;
+      color: $link-color;
```

Comment:
@DaleMcGrew side comment: I'm wondering what an appropriate default way to style link text on the site is since I'm noticing js handlers seem to be used to capture most click events... The simplest would be to still wrap links in `<a>` tags. Does that seem reasonable?